pkg/cloud/amazon/vault/vault_resources.go

```diff
@@ -370,29 +449,68 @@ func runCloudformationTemplate(templateBody *string, stackName *string) error {
 		Capabilities: []*string{
 			aws.String("CAPABILITY_NAMED_IAM"),
 		},
+		Parameters: parameters,
 	}
 
 	_, err := svc.CreateStack(input)
 	if err != nil {
-		return errors.Wrapf(err, "Unable to create vault preresiquite resources")
+		return errors.Wrapf(err, "Unable to create vault prerequisite resources")
 	}
-
+	donec := make(chan bool)
+	go runStackEventsReporting(donec, svc, stackName)
 	// Wait until stack is created
 	err = svc.WaitUntilStackCreateComplete(desInput)
```

Comment:
Does this wait has any timeout in case the stack creation gets stuck? It would be nice to bail out after a while.

Reply:
The default is 120 attempts with a 30 seconds of delay, this results in a total timeout of 1h.

Both attempts and delay can be modified. What would you think it's a good time to wait?